modulefiles/ufs_common.lua

```diff
@@ -30,3 +30,5 @@ for i = 1, #ufs_modules do
     load(pathJoin(name, os.getenv(env_version_name) or default_version))
   end
 end
+
+load("nco")
```

Comment:
Does nco available in all the platforms?

Reply:
I don't have access to all platforms to check... @jkbk2004 ?

Reply:
@RatkoVasic-NOAA do we have nco on all the R&D platforms?

Reply:
> @RatkoVasic-NOAA do we have nco on all the R&D platforms?

Yes:
```
Hera:
------------------------------------------------------------------------------------------------- /apps/modules/modulefiles -------------------------------------------------------------------------------------------------
   nco/5.1.6 (D)
Jet:
---------------------------------------------------------------------- /apps/modules/modulefiles ----------------------------------------------------------------------
   nco/5.1.6
Gaea:
-------------------------- /sw/gaea-c5/spack-envs/base/modules/spack/linux-sles15-x86_64/Core --------------------------
   nco/4.7.9    nco/5.0.1 (D)

Hercules:
---------------------------------------------- /apps/spack-managed/modulefiles/linux-rocky9-x86_64/Core -----------------------------------------------
   nco/5.0.1    nco/5.1.6 (D)    w3nco/2.4.1
Orion:
------------------------------------------------------------------- /apps/modulefiles/compilers/intel-2018 --------------------------------------------------------------------
   nco/4.8.1    nco/4.9.3 (D)
Derecho:
------------------------------------------------------------------------------ Compilers and Core Software ------------------------------------------------------------------------------
   nco/5.1.6    nco/5.1.9 (D)

```

Reply:
derecho needed load("nco") in its modulefiles as well. I don't know what to do about platforms:
expanse linux macosx odin stampede

Reply:
Update:
I prefer a temporary `module load nco` locally in tests/fv3_conf/cpld_docn_dice.IN.
load("nco") in modulefiles is not enough on derecho to find ncrcat.

Once bad histaux output (https://github.com/ESCOMP/CMEPS/issues/430) is resolved, many timesteps of component-->mediator couplings can be directly output to one file.

An alternate solution is to reconfigure dxxx.streams to specify the separate mediator history filenames over time, but formatting is error prone